src/functions-reference/matrix_operations.Rmd

```diff
@@ -2278,6 +2300,17 @@ implies that the resulting matrix is square with the same number of
 columns as A
 `r since("2.18")`
 
+
+<!-- tuple(matrix, matrix); qr_thin; (matrix A); -->
+\index{{\tt \bfseries qr\_thin }!{\tt (matrix A): tuple(matrix, matrix)}|hyperpage}
+
+`tuple(matrix, matrix)` **`qr_thin`**`(matrix A)`<br>\newline
+Returns both portions of the QR decomposition of A. This function
```

Comment:
This should mention that Q is orthonormal (in the column vectors) and R is upper triangular.